db/views.sql

```diff
@@ -0,0 +1,30 @@
+SELECT * FROM type;
+SELECT * FROM type WHERE name LIKE '%water%';
+SELECT * FROM typematchup;
+
+SELECT * FROM pokemon ORDER BY nationalDexNumber;
+
+SELECT * FROM pokemon
+JOIN type 
+where pokemon.name LIKE "%Blastoise%";
+
+SELECT 
```

Comment:
Added a one-line comment above this line describing what the `SELECT` will perform. Do the same for the other views.